doc/requirements.txt

```diff
@@ -20,3 +20,4 @@ m2r2
 mistune==0.8.4
 sphinxext-opengraph==0.9.0
 matplotlib==3.8.0
+tomlkit
```

Comment:
We don't need to add this here when it's already added to `setup.py` as a package dep
```suggestion
```